apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.ejs

```diff
@@ -0,0 +1,326 @@
+					<div class="modal-content">
+						<div class="modal-header">
+							<h5 class="modal-title" id="addPolicyModalLabel">Add New Override</h5>
+							<button type="button" class="close" data-dismiss="modal" aria-label="Close">
+								<span aria-hidden="true">&times;</span>
+							</button>
+						</div>
+						<div class="modal-body">
+							<form id="add-new-override" method="POST">
+								<input type="hidden" name="__action" value="add_new_override">
+								<input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+								<div class="form-group">
+									<!-- <label for="assessment_id">Assessment ID</label> -->
+									<input type="hidden" class="form-control" id="assessment_id" name="assessment_id" value="<%= assessment_id %>" required readonly>
+								</div>
+								<div class="form-group">
+									<label for="student_uid">Student UID</label>
+									<input type="text" class="form-control" id="student_uid" name="student_uid">
+								</div>
+								<div class="form-group">
+									<label for="group_id">Group ID</label>
+									<input type="number" class="form-control" id="group_id" name="group_id">
+									<div id="group_id_error" class="invalid-feedback" style="display: none; color: red;"></div>
+								</div>
+								<div class="form-group">
+									<label for="created_at">Created At</label>
+									<input type="datetime-local" class="form-control" id="created_at" name="created_at" required readonly>
+								</div>
+								<div class="form-group">
+									<label for="start_date">Start Date</label>
+									<input type="datetime-local" class="form-control" id="start_date" name="start_date" required>
+								</div>
+								<div class="form-group">
+									<label for="end_date">End Date</label>
+									<input type="datetime-local" class="form-control" id="end_date" name="end_date" required>
+									<div id="end_date_error" class="invalid-feedback" style="display: none; color: red;"></div>
+								</div>
+								<div class="form-group">
+									<!-- <label for="created_by">Created By</label> -->
+									<input type="hidden" class="form-control" id="current_student_uid" name="current_student_uid" value="<%= current_student_uid %>" required readonly>
+								</div>
+								<div class="form-group">
+									<label for="credit">Credit</label>
+									<input type="number" class="form-control" id="credit" name="credit">
+								</div>
+
+
+								<div class="form-group">
+									<label for="note">Note</label>
+									<textarea class="form-control" id="note" name="note"></textarea>
+								</div>
+
+								<div class="form-group">
+									<label for="type">Type</label>
+									<select class="form-control" id="type" name="type" required>
+										<option value="">Select a type</option>
+										<option value="manual">Manual</option>
+										<option value="extension_token">Extension Token</option>
+									</select>
+								</div>
```

Comment:
IMO, we could skip this for now and remove the corresponding database column / backend code. There's little point putting "extension token" in the UI/DB at this point.